### PR TITLE
docs:Add svelte js

### PR DESCRIPTION
**Pull request by getspooky**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -178,6 +178,7 @@ A collection of awesome browser-side [JavaScript](https://developer.mozilla.org/
 * [meteor](https://github.com/meteor/meteor) - An ultra-simple, database-everywhere, data-on-the-wire, pure-javascript web framework.
 * [ractive](https://github.com/ractivejs/ractive) - Next-generation DOM manipulation.
 * [vue](https://github.com/vuejs/vue) - Intuitive, fast & composable MVVM for building interactive interfaces.
+* [svelte](https://github.com/sveltejs/svelte) - Svelte is a tool for building fast web applications. It is similar to JavaScript frameworks such as React and Vue, which share a goal of making it easy to build slick interactive user interfaces.
```
Comment (donnisnoni95) [inline README.md]: How about this ??

> "Svelte is a new way to build web applications. It's a compiler that takes your declarative components and converts them into efficient JavaScript that surgically updates the DOM."

Comment (gokaygurcan) [reply README.md]: This is their own description: https://github.com/sveltejs/svelte#what-is-svelte
I'd go for this, indeed. Thanks!

Comment (donnisnoni95) [reply README.md]: ok.. i appreciate that... good job !!!

**Comment by marconi1992** [comment]
> Please move newly added links at the end of their block. Thanks!

Hi @getspooky. Could you do the requested change from @gokaygurcan? I think this ticket is important to merge.

**Comment by getspooky** [comment]
Change svelte description

**Pull request merged**